index.js

```diff
@@ -192,16 +193,46 @@ checker.init({ start: program.path, production: true, customFormat: customFormat
           let licenseFile = license.licenseFile.toLowerCase();
           return licenseFile.indexOf('not found') !== -1 || licenseFile.indexOf('readme') !== -1;
         });
-
-        let {fixedLicenses, fixesLog} = await fixLicenseErrors(licenseErrors, program.auth);
-        includeFixes(licenses, fixedLicenses);
+        console.log(`${licenseErrors.length} packages found with incorrect licenses.`);
+
+        let shouldFixLicenses = licenseErrors.length && await (async () => {
```

Comment:
I usually run this script as part of a Jenkins build. I assume that this'll block. Can we add a command line option to disable this?

Reply:
Sure, Andrew. I will add those options.

Reply:
Andrew, I have added this feature. Please have a look.